### DISCUSSION TITLE
Disable Object expiration

### DISCUSSION THREAD

**Discussion by KnightKrusty** [minio/minio#12625] [Q&A]
I want to disbale the default 5 day expiration for My object in the bucket, I am not able to find any config option related to that.

**Comment by klauspost** [comment]
@KnightKrusty Which "default 5 day expiration" are you referring to?

**Reply by KnightKrusty** [reply]
When you click on share button on webUI in Minio Browser? is there another way to get url of uploaded object?

**Reply by harshavardhana** [reply]
That's not allowed
